web/src/lib/data-hooks/useFormContextFieldHelpers.ts

```diff
@@ -65,10 +65,10 @@ export const useFormContextFieldHelpers = <T, FieldError = FieldErrorType>(
   // eslint-disable-next-line react-hooks/rules-of-hooks
   const isFormFieldInValid = useMemo(
     () => {
-      return fieldStates[fieldName].invalid === true;
+      return fieldStates[fieldName]?.invalid === true;
```

Comment:
Why did we need to make this change?

I get wary when we start modifying the underlying data-hooks just because how much we use these throughout the application.